analyzers/dataframe/src/JetConstituentsUtils.cc

```diff
@@ -1068,6 +1068,36 @@ namespace FCCAnalyses {
       return std::sqrt(E*E - px*px - py*py - pz*pz);
     }
 
+    rv::RVec<double> AllInvariantMasses(rv::RVec<TLorentzVector> AllJets) {
```

Comment:
Ca you use `snake_case` for the name? Just to be consistent with other functions.

Reply:
Yes - in that case shall I also update `InvariantMass` --> `invariant_mass` ?

Reply:
I do not like `InvariantMass` either, but I would probably leave it as it is since changing might break people's working code.

Reply:
@selvaggi - ok, I left `InvariantMass` as is